### PR TITLE
Add v1 of regionally calibrated TC functions

### PR DESCRIPTION
**Pull request by ChrisFairless**
Here are the first calibrated TC economic loss impact functions.

Looking at first numbers, they roughly match what you've seen with the regionally calibrated functions. All calibrations result in higher losses, which is probably good for our interim results but we'll want to make sure that's correct.

In particular, we'll see numbers jump in North Indian and Central America/Caribbean regions, plus a couple others.

The number for the Rest of the World is just a guesstimate: the calibration for this takes hours and wasn't finished. It's lower than most curves and that means that Ireland is completely enormous in losses. Once again: ignore europe with TCs 😭

### CODE REVIEW

**Review by mastaia — CHANGES_REQUESTED** [review]
Add the possibility to switch on and off the use of this calibrated functions

**Comment by mastaia** [comment]
Add the changes to the config

**Comment by ChrisFairless** [comment]
@mastaia  I've added these changes: there are two new settings in a config file:
`apply_business_interruption`:  turn off to assume % asset loss = % production loss
`use_calibrated_impfs`: turn off to use unclibrated regional best guesstimate impact functions

I'm not sure I like this – to implement it I've had to add `business_interruption` and `calibrated` parameters to loads of functions so that the infomation can be passed through the pipeline...

But maybe we're overthinking at this point.

**Comment by ChrisFairless** [comment]
I'll merge this for now.

At some point in the future we might want to simplify how we pass modelling parameters around. It feels really clunky to add these `business_interruption` and `calibrated` parameters to so many function calls, but I don't have a great alternative. I guess they could be environment variables, but that's also an extra layer of complexity and something that the user has to keep track of.
